cli/run_util.go

```diff
@@ -617,14 +617,39 @@ func activateAndRunSteps(
 			errStr = err.Error()
 		}
 
+		stepInputs := make(map[string]string)
```

Comment:
pls move this chunk into a new function, something like:

`func expandStepInputs(inputs []env, configEnvs []env) (expandedStepInputs []envs, error)`

Reply:
once you have the function pls add unit test, some suggested cases:

0. case: test if input gets expanded from configEnvs
1. case: test if input gets expanded from os.environ
2. case: test if expanding happens across inputs
3. make sure expand order is the same as envman does